annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/AnnotationServiceImpl.java

```diff
@@ -671,24 +729,107 @@ private void validateTranscriptionWithFullTextResource(Body body)
 	if (StringUtils.isBlank(body.getEdmRights())) {
 	    throw new PropertyValidationException(ParamValidationException.MESSAGE_MISSING_MANDATORY_FIELD,
 		    I18nConstants.MESSAGE_MISSING_MANDATORY_FIELD, new String[] { "transcription.body.edmRights" });
-	}
+    }
 	validateEdmRights(body);
     }
+    
+    private void validateTagWithFullTextResource(Body body) throws ParamValidationException, PropertyValidationException {
```

Comment:
Please remove this method, we don't support such use case

Reply:
Method removed